astroplan/tests/test_observer.py

```diff
@@ -1156,3 +1157,33 @@ def vectorize_timing(n_targets):
     obs.target_rise_time(t, target_list)
     end = time()
     return end-start
+
+def test_local_sidereal_time():
+    time = Time('2005-02-03 00:00:00')
+    location = EarthLocation.from_geodetic(10*u.deg, 40*u.deg, 0*u.m)
+    obs = Observer(location=location)
+    # test sidereal time
+    astroplan_lst = obs.local_sidereal_time(time)
+    # Compute this with print_pyephem_lst()
+    pyephem_lst = 2.5005375428099104*u.rad
+    assert_quantity_allclose(pyephem_lst, astroplan_lst, atol=0.01*u.deg)
```

Comment:
```
Signature: assert_quantity_allclose(actual, desired, rtol=1e-07, atol=None, **kwargs)
```
So I'd say here, `actual=astroplan_lst` and `expected=pyephem_lst`, i.e. exchange the order of the arguments
to get a clearer message should it fail somewhere.

(Maybe have a look around checking that this is done correctly throughout astroplan?)